src/governance-object.h

```diff
@@ -25,6 +25,7 @@ class CGovernanceVote;
 
 static const int MAX_GOVERNANCE_OBJECT_DATA_SIZE = 16 * 1024;
 static const int MIN_GOVERNANCE_PEER_PROTO_VERSION = 70204;
+static const int GOVERNANCE_FILTER_PROTO_VERSION = 70206;
```

Comment:
I tend to disagree. Why would I want to search for bloom filter/protocol version consts in governance _object_ header when these clearly do not belong there? New bloom constants should not even be used anywhere besides `CGovernanceManager` at all.
Imo introducing them in `governance-object.h` is confusing.